src/PolykeyClient.ts

```diff
@@ -60,15 +62,17 @@ class PolykeyClient {
       logger: logger.getChild(Session.name),
       fresh,
     });
-    const rpcClientClient = await RPCClient.createRPCClient({
+    const rpcClientClient = new RPCClient({
       manifest: clientManifest,
       streamFactory,
       middlewareFactory: rpcUtilsMiddleware.defaultClientMiddlewareWrapper(
         clientUtilsMiddleware.middlewareClient(session),
         parserBufferByteLimit,
       ),
+      toError: networkUtils.toError,
```

Comment:
I thought `toError` here would be specific to client domain? Or are we using `network/utils` for both client service and agent service?

Reply:
network domain is for stuff shared between client and agent network stuff.  One thing that still needs to be done is the `fromError` used for the clientService  needs to be a sensitive version that strips out the callstack.

Reply:
The sensitive version is just whatever the `replacer` is. So I don't think `fromError` does any stripping. You define the `replacer` and that should be one stripping out the call stack and any other relevant information.

Reply:
`fromError` can do the stripping. Why do we even need the replacer? In any case, the `fromError` and `toError` needs to be fixed up. I didn't want to spend much more time on it with other things needing fixes at the time.

I think @amydevs should have a look over it.

Reply:
It's cause `fromError` returns a JSON object to be stringified. The `replacer` can only be used when you do `JSON.stringify`. It's 2 steps.

Reply:
New issue at #564 to address this.